droidlet/tools/hitl/simple_hitl_example.py

```diff
@@ -0,0 +1,121 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+
+This is a simple example demonstrating the usage of HiTL library
+
+It registers:
+1. a FileWriter DataGenerator which will generate certain kind of file to your local FS
+2. a FileListener JobListener which will listen to your local FS for files FileWriter generated
+   and perform in-place transformation of them -- overwriting the content of them.
+"""
+
+import logging
+import os
+import re
+import time
+
+from typing import List
+
+from .data_generator import DataGenerator
+from .job_listener import JobListener
+from .task_runner import TaskRunner
+
+
+WORKDIR = "/private/home/yuxuans/Workspace/tmp/droidlet/droidlet/tools/hitl/tmp/"
```

Comment:
Might not want your actual filepath committed to main

Reply:
Agree with this, lets not add hardcoded paths

Reply:
resolved.